docs/content/manual/_incoming/1707-zero-downtime-upgrade.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Zero downtime upgrade 
+---
+
+* Related issues: [#1707](https://github.com/harvester/harvester/issues/1707) [BUG] Zero downtime upgrade stuck in "Waiting for VM live-migration or shutdown..."
+
+## Category: 
+* Upgrade
+
+## Verification Steps
```

Comment:
Do we want to test the scenario where upgrade failed and Harvester keeps working with older version. VMs are still working.

Reply:
In the upgrade experience of the previous v1.0.1 - v1.0.3 release.  

Development team recommend to shutdown all VM before upgrade to reduce the impact of upgrade failure. 
But typically I perform the upgrade test on recent release without shutting down VM to test the availability.  

When the upgrade not yet enter the `pre-drain` step in Phase 4 (to migrate VM) 
![image](https://user-images.githubusercontent.com/29251855/186045284-b3077719-1f45-4eb8-838d-e586a883d35f.png)

If the upgrade failed prior to pre-drain, the VM could be able to maintain it's status 
If failed after the pre-drain step, the VM may be affected. 

It worth having a negative case of it. I would add it later by adding some v1.0.3 manual test plan